shared/src/business/entities/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { v4 as uuidv4 } from 'uuid';
+
+export const getUniqueId = (): string => {
+  return uuidv4();
+};
```

Comment:
We do have [an existing function](https://github.com/ustaxcourt/ef-cms/blob/a99e10c3de7eb7cb0eed1fc7ca86eea8d83f8df5/shared/src/sharedAppContext.ts#L12). Maybe we use one or the other?

Reply:
Will do.